pages/playground.js

```diff
@@ -140,19 +141,7 @@ export default function Playground() {
           }}
         />
         <div className="flex flex-row justify-end mb-3 gap-2">
-          {!formatComplete && (
-            <Button text="Format" onClick={handleFormatJson} primary={false} />
-          )}
-          {formatComplete && !validateComplete && (
-            <Button
-              text="Validate"
-              onClick={handleValidateJson}
-              primary={false}
-            />
-          )}
-          {formatComplete && validateComplete && (
-            <Button text="Preview" onClick={handlePreview} primary={true} />
-          )}
+          <Button {...(!formatComplete ? {text: "Format", onClick: handleFormatJson, primary: false} : formatComplete && !validateComplete ? {text: "Validate", onClick: handleValidateJson, primary: false} : formatComplete && validateComplete ? {text: "Preview", onClick: handlePreview, primary: true} : {text: "", disable: true})} />

```

Comment:
Please check the formatting, this is not readable. If you have the VScode Prettier plugin installed, this should pick up our settings and do it automatically for you

Reply:
Sorry,
I have Prettier disabled for workspaces.
Pushed formatting changes.